sw/device/tests/power_virus_systemtest.c

```diff
@@ -578,14 +587,39 @@ static void max_power_task(void *task_parameters) {
 
   // Toggle GPIO pin to indicate we are in max power consumption.
   CHECK_DIF_OK(dif_gpio_write(&gpio, 0, true));
+  // TODO (#14814): confirm AES is the fastest throughput cryto block.
+  // Otherwise, wait for that block to finish processing instead.
```

Comment:
I did a little digging on this one and AES should indeed be the fastest. According to each block's docs:
[aes](https://docs.opentitan.org/hw/ip/aes/doc/#features):
> * Latency per 16 byte data block of 12/14/16 clock cycles (unmasked implementation) and 56/66/72 clock cycles (DOM) in AES-128/192/256 mode

[kmac](https://docs.opentitan.org/hw/ip/kmac/doc/#keccak-round):
> Processing a _Keccak_f_ (1600 bit state) takes a total of 96 cycles (24 rounds X 4 cycles/round) including the 1st and 2nd phases.

[hmac](https://docs.opentitan.org/hw/ip/hmac/doc/#performance-in-sha-256-mode-and-hmac-mode):
> ...it takes 80 cycles to complete a block. The effective throughput considering this is 64 byte / 80 clk or 16 clk / 80 clk, 20% of the maximum throughput. For instance, if the clock frequency is 100MHz, the SHA-256 can hash out 80MB/s at most...If HMAC mode is turned on, it introduces extra latency due to the second round of computing the final hash of the outer key and the result of the first round using the inner key. This adds an extra 240 cycles (80 for the inner key, 80 for the outer key, and 80 for the result of the first round) to complete a message. For instance, if an empty message is given then it takes 360 cycles (80 for msg itself and 240 for the extra) to get the HMAC authentication token.

When all three are triggered, we're waiting for:
- One block to be processed by AES (72 cycles)
- One `Keccak-f` from KMAC (96 cycles)
- Inner + outer key computation from HMAC (> 240 cycles)

Reply:
Ah thank you @jadephilipoom, this is wonderful, exactly was I was looking for!

I will confirm via the waveform as well, and add some assertions (in a follow up PR) to ensure the test fails loudly in case these ever change, and we need to change the test to accomodate.